week-1/interpret/percentage-change.js

```diff
@@ -12,9 +12,13 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// Ans. 4 calls in line 4 and 5 
```

Comment:
There's one more function call I can spot in the code as well

Reply:
If I were to take a wild guess now I would say that console.log is the other function call?